scripts/backend.ps1

```diff
@@ -1,6 +1,11 @@
 ##### Install PowerShell 5 using https://github.com/DarwinJS/ChocoPackages/blob/master/PowerShell/v5.1/tools/ChocolateyInstall.ps1#L107-L173
```

Comment:
Change commit message to:
Import the common_functions at the start of the backend.ps1

This is needed for a better code style and for the unit tests to pass.